### DISCUSSION TITLE
如何在html_sql中判断vec长度>0

### DISCUSSION THREAD

**Discussion by mr-daijunchao** [rbatis/rbatis#271] [General]
<if test="sorts!= null && sorts.length>0">
                order by
                <foreach collection="sorts" index="key" item="item" separator=",">
                    #{item}
                </foreach>
            </if>

length不行啊

**Comment by mr-daijunchao** [comment]
![image](https://user-images.githubusercontent.com/96764154/155663227-7c194bcb-cab5-4e18-961a-ffd2ad69cacb.png)

**Comment by zhuxiujia** [comment]
> order by #{item}
> length不行啊

* foreach 外面套一层if
```xml
<if>

<if test="sorts.len()>0">
<foreach collection="sorts>
</foreach>
</if>
</if>
```
